libcontainer/factory_linux.go

```diff
@@ -277,7 +277,7 @@ func (l *LinuxFactory) loadState(root string) (*State, error) {
 	f, err := os.Open(filepath.Join(root, stateFilename))
 	if err != nil {
 		if os.IsNotExist(err) {
-			return nil, newGenericError(err, ContainerNotExists)
+			return nil, newGenericError(fmt.Errorf("container not exists"), ContainerNotExists)
```

Comment:
Can you add the container's id to the error message? 

`"container %q does not exist"`